sharing/README.md

```diff
@@ -25,3 +25,172 @@ High level features include:
+
+### 1. Selecting an Organization
+
+The first step involves deciding which organizations a rule applies to. This is done using the `sharedWith` column. A unique identifier for each organization should be used and reused throughout the entire document. The different organizations that a rule pertains to should be seperated by a ";". For example, if a rule applies to the **Public Health Agency of Canada** (PHAC) as well as **Ottawa Public Health** (OPH) the value of the `sharedWith` cell in the row for that rule would be `PHAC;OPH`. The example assumes that PHAC and OPH are the agreed upon identifiers to represent these organizations.
+
+### 2. Selecting an Entity
+
+The second step involves selecting the parts of the ODM data model or entities that the rule applies to. The entities that can be selected are:
+
+* Data contained in a table
+* Data contained in column(s) of table(s)
+
+This step uses two columns, `tableName` and `variableName`. The `tableName` column can specify the name(s) of the table this rules applies to and the `variableName` column can specify the name(s) of the columns this rule applies to.
+
+Some examples are given below:
+
+1. Selecting the `type` column in the `Sample` table
+
+    | tableName | variableName |
+    |-----------|--------------|
+    | Sample    | type         |
+
+2. Selecting the `qualityFlag` and the `type` column in the `WWMeasure` table
+
+    | tableName        | variableName        |
+    |------------------|---------------------|
+    | WWMeasure        | qualityFlag;type    |
+
+3. Selecting all the columns in the `WWMeasure` table
+
+    | tableName        | variableName |
+    |------------------|--------------|
+    | WWMeasure        | all          |
+
+4. Selecting a the `type` column in the `WWmeasure` and the `Sample` table
+
+    | tableName               | variableName  |
+    |-------------------------|---------------|
+    | WWMeasure;Sample        | type          |
+
+5. Selecting the `type` column in all tables
+
+    | tableName               | variableName  |
+    |-------------------------|---------------|
+    | all                     | type          |
+
+Some things to note:
+
+* In examples 2 and 4 where multiple columns and tables were selected respectively, a **;** was used to seperate the values. The same symbol was used to seperate multiple organizations in the previous step. In fact, throughout the entire document when multiple values need to put into a cell, the **;** symbol should be used to seperate them.
+* In examples 3 and 5 where all the columns in a table and all the tables were selected respectively, the keyword **all** was used. Similar to the **;** symbol, the keyword **all** may be used in a cell to mean everything.
+
+### 3. Selecting the Direction of the Rule
```

Comment:
I like this concept but I don't see how it is implemented below. 

Generally, in data science we think of:
`filter` --> column 
`select` --> row. 

What about changing this terms from `direction` to `filter and select`?

Reply:
I'm going to backtrack on my suggestion. The 'direction' of the rule could work well. It is simple and inuitive.